user-manual/process_screenshots.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+pushd $SCRIPT_PATH
+
+mkdir -p output/setup/img/
+cp screenshots/welcome_screen.png output/setup/img/
+
+mkdir -p output/accounts/img/
+
+cp screenshots/account_setup_* output/accounts/img/
+cp screenshots/drawer_account_list.png output/accounts/img/
+
+# Adding another account
+convert screenshots/drawer_one_account.png -stroke red -strokewidth 5 -fill none -draw "rectangle 20,1750 820,1840 " output/accounts/img/drawer_settings_highlight.png
```

Comment:
We should mention that ImageMagick is required for this to work.

Reply:
I added a README with requirements and usage instructions.